src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -294,6 +294,7 @@ def _encode_prompt(
         negative_prompt=None,
         prompt_embeds: Optional[torch.FloatTensor] = None,
         negative_prompt_embeds: Optional[torch.FloatTensor] = None,
+        lora_scale: Optional[float] = None,
```

Comment:
I am struggling to understand why do we need to pass this to `_encode_prmompt()`. 

LoRA scale for the text encoder is applied after the LoRA weights for the text encoder is loaded.

Reply:
@patrickvonplaten if we do this then we cannot have `# copied from ...` statements in the other pipelines as you can see on the CI.